app/views/pages/privacy.html.haml

```diff
@@ -0,0 +1,30 @@
+- content_for :page_title do
+  Privacy Policy
+
+%h1.dividing.header Privacy Policy
+%p
+  Privacy takes the front stage in transientBug. That being said, don't store truly sensitive information on transientBug.
+
+%p
+  All user data is stored on transientBug servers, and the official web extensions only communicate with the transientBug API over an HTTPS connection.
+
+%p
+  The following data is sent by the official web extension when its page action is clicked: The active tabs Title and URL. That's it.
+
+%h2.dividing.header Backups and Account deletion
+%p
+  transientBug does backup it's database to help prevent catastrophic disasters (And because I'd be really sad if my own site lost my own bookmarks). At the moment these backups are manual and happen once a week or so simply because the traffic to the site is so low. These backups are stored in a secured, private Digital Ocean Space and deleted after the next backup takes place. I am looking into setting up an automatic backup system that will keep the last 7 days of backups, taken twice a day (UTC noon and midnight). Anything older than 7 days will automatically be destroyed by this system as well.
+%p
+  If you request your account to be closed, all your bookmarks, authorized apps, and developer apps are destroyed. Gone forever, at least unless I'm forced to restore a database backup. Due to the nature of backups, its highly difficult to remove data from them until after they're restored, however I'll make my best attempt at re-closing your account if this happens.
+
+%h2.dividing.header Additional details
+%p
+  %ul.list
+    %li Privacy related bugs take priority.
+    %li Everything is under SSL, including the API for extensions and apps.
+    %li No ads, ever.
+    %li No analytic scripts, third party data collection.
+    %li I will never share, sell or otherwise give away user data to third parties.
+    %li If there is a privacy breach, all users will be notified via the appropriate channels.
+    %li If you find a privacy leak, please be responsible and report it discreetly to: me (at) transientbug.ninja
```

Comment:
`discreetly` the term most of the security community uses is "responsibly"

Reply:
I already said responsible too :P